dex/src/main/scala/com/wavesplatform/dex/api/http/routes/MatcherApiRoute.scala

```diff
@@ -1168,7 +1165,7 @@ class MatcherApiRoute(
   )
   def getMatcherConfig: Route = (path("config") & get & withAuth) {
     complete {
-      HttpEntity(filteredConfig.rendered).withContentType(CustomContentTypes.`application/hocon`)
+      HttpEntity(config.rendered).withContentType(CustomContentTypes.`application/hocon`)
```

Comment:
Consider rename `config` in the constructor to `safeConfig` to make things clear. Otherwise someone might use `config` and think that it contains all keys.

Reply:
+